tests/test_flows/test_flow.py

```diff
@@ -153,6 +153,23 @@ def test_publish_flow(self):
         flow.publish()
         self.assertIsInstance(flow.flow_id, int)
 
+    def test_semi_legal_flow(self):
+        # TODO: Test if parameters are set correctly!
+        sentinel = get_sentinel()
+        semi_legal = sklearn.ensemble.BaggingClassifier(
+            base_estimator=sklearn.ensemble.BaggingClassifier(
+                base_estimator=sklearn.tree.DecisionTreeClassifier()))
+        flow = openml.flows.sklearn_to_flow(semi_legal)
+        flow.name = 'TEST%s%s' % (sentinel, flow.name)
+
+        flow.publish()
+
+    def test_illegal_flow(self):
```

Comment:
Could you please add a docstring why exactly this is illegal? Someone not too familiar with OpenML might not know about this.